test/potluck_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/potluck'
+require './lib/dish'
+
+class PotluckTest < Minitest::Test 
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+    
+    assert_instance_of Potluck, potluck 
+  end 
+  
+  def test_it_has_a_date
+    potluck = Potluck.new("7-13-18")
+    
+    assert_equal "7-13-18", potluck.date 
+  end
+  
+  def test_dishes_start_with_empty_array
+    potluck = Potluck.new("7-13-18")
+    
+    assert_equal [], potluck.dishes 
+  end 
+  
+  def test_it_adds_dish
```

Comment:
In this test, you want to assert `couscous_salad` is what is in `potluck.dishes` _after_ it has been added to the potluck.  It should look something like this:
```ruby
def test_it_add_a_dish
  potluck = Potluck.new("7-13-18")
  couscous_salad = Dish.new("Couscous Salad", :appetizer)
  potluck.add_dish(couscous_salad)
  assert_equal couscous_salad, potluck.dishes[0]
end